0000-template.md

```diff
@@ -0,0 +1,47 @@
+- Feature Name: (fill with a unique ident, my_awesome_feature)
+- Start Date: (fill with today's date, DD-MM-YYYY)
+- RFC PR: (leave this empty)
```

Comment:
```suggestion
```

Reply:
I find it very usefull to have the pull request number in the rfc.
With this you can find the discussion also years later.

Reply:
Will keep PR number